frontend/src/app/admin/superadmin/structures/structures.component.scss

```
@@ -65,52 +65,124 @@
-      font-size: 14px;
-    }
+.structure-title {
+  min-width: 0;
+
+  h3 {
+    margin: 0;
+    color: var(--ink-900);
+    font-size: 1.15rem;
+    font-weight: 700;
+    line-height: 1.25;
+    word-break: break-word;
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)